platform/admin/core/admin-core-logviewer/pom.xml

```diff
@@ -78,6 +83,7 @@
                 <artifactId>maven-bundle-plugin</artifactId>
                 <configuration>
                     <instructions>
+                        <Embed-Dependency>platform-error-servlet</Embed-Dependency>
```

Comment:
Is there any reason platform-error-servlet is a bundle? It doesn't export anything and everyone is embedding it already